scanners/ios/podfile.go

```diff
@@ -111,21 +113,21 @@ func (podfileParser podfileParser) shouldRaiseReadDefinitionError(err string) bo
 	return true
 }
 
-func (podfileParser podfileParser) getUserDefinedProjectRelavtivePath(cocoapodsVersion string) (string, error) {
+func (podfileParser podfileParser) getUserDefinedProjectAbsPath(cocoapodsVersion string) (string, error) {
 	targetProjectMap, err := podfileParser.getTargetDefinitionProjectMap(cocoapodsVersion)
 	if err != nil {
 		return "", fmt.Errorf("failed to get target definition map: %s", err)
 	}
 
-	for target, project := range targetProjectMap {
-		if target == "Pods" {
-			return project, nil
-		}
+	// Return the first custom project
```

Comment:
Fixed/worked around a problem that only top-level (Pod target) projects were parsed from Podfiles. Now using the first project, disregarding which target is it in.

# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'Sample' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!
  project 'Sample/Sample.xcodeproj'
  # Pods for Sample
  pod 'Alamofire', '~> 5.5'
end